onset_maproom/calc.py

```diff
@@ -51,6 +51,37 @@ def water_balance(
     return water_balance
 
 
+def following_dry_spell_length(daily_rain, wet_thresh, time_coord="T"):
+    """Computes the count of consecutive dry days (or dry spell length) after each day
+    Ideally we would want to count dry days backwards
+    and reset count to 0 each time a wet day occurs.
+    But that is hard to do vectorially.
+    But we can count all dry days backwayds then apply an offset, in more details:
+    Cumul dry days backwards to get all dry days after a day
+    Find when to apply new offset (dry days followed by a wet day)
+    Assign cumulated dry days there, Nan elsewhere
+    Propagate backwards and the 0s at the tail
+    And that is going to be the offset
+    Apply offset that is correct for all days followed by dry days
+    Eventually reset days followed by wet days to 0
+    """
+    # Find dry days
+    dry_day = ~(daily_rain > wet_thresh) * 1
+    # Cumul dry days backwards and shift back to get the count to exclude day of
+    dry_spell_length = dry_day.reindex({time_coord: dry_day[time_coord][::-1]}).cumsum(
```

Comment:
Give this variable a better name. It's only an intermediate step on the path to calculating dry spell length, one of many, and the others all have meaningful names.

Reply:
I am pretty conservative on naming inflation after 10+ years of rolling stacked objects... I see I can change my ways...